### ISSUE TITLE
[iOS-Android][Markdown rendering parity] [Bulleted list is not in parity with desktop]

### ISSUE THREAD

**Issue by adumrewal** [microsoft/AdaptiveCards#4311]
# Platform

What platform is your issue or question related to? (Delete other platforms).

* Android
* iOS
* UWP

# Author or host

Host - Teams

# Version of SDK

Latest master

# Details

For the following cards JSON, desktop and mobile have different experiences.
|Mobile|Desktop|
|:-:|:-:|
|<img width="250" alt="Screenshot 2020-07-07 at 5 56 19 PM" src="https://user-images.githubusercontent.com/8917620/86781416-7c787180-c07b-11ea-8b66-ec9af66475da.png">|<img width="250" alt="Screenshot 2020-07-07 at 5 56 54 PM" src="https://user-images.githubusercontent.com/8917620/86781459-8a2df700-c07b-11ea-88a8-bafbc61ec3e0.png">|

```
{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.2",
  "body": [
    {
      "type": "TextBlock",
      "color": "warning",
      "text": "What time do you want to meet?   \n* Hello   \n* Item 2   \n* Item 3 ",
      "wrap": true
    }
  ],
  "actions": [
    {
      "type": "Action.Submit",
      "title": "OK"
    }
  ]
}
```

**Comment by jwoo-msft** [comment]
@adumrewal 
I can't reproduce the issue. I tested it on both main and 1.2.10 iOS branch.
![image](https://user-images.githubusercontent.com/4112696/86875181-c458c200-c096-11ea-9de2-f876a0e7a96f.png)

**Comment by jwoo-msft** [comment]
This is already fixed via PR attached to #4117. FYI, We migrated to the main branch from the master.
